nodes/filter/nodes.go

```diff
@@ -0,0 +1,225 @@
+	outputs := node.BuildOutputs(node.BuildOutput(node.Out, node.TypeFloat, nil, body.Outputs))
+	body = node.BuildNode(body, inputs, outputs, nil)
+	body.SetHelp(fmt.Sprintln("This node filters ‘input’ values. All ‘input’ values are passed to ‘output’ EXCEPT ‘input’ values which are equal to ‘match’. Output the last known input float value if two inputs equal to each other. Out put nil if any of the inputs are nil."))
+	return &PreventEqualFloat{body, 0}, nil
+}
+
+func NewPreventEqualString(body *node.Spec) (node.Node, error) {
+	body = node.Defaults(body, preventEqualString, category)
+	inputs := node.BuildInputs(node.BuildInput(node.In, node.TypeString, nil, body.Inputs), node.BuildInput(node.InputMatch, node.TypeString, nil, body.Inputs))
+	outputs := node.BuildOutputs(node.BuildOutput(node.Out, node.TypeString, nil, body.Outputs))
+	body = node.BuildNode(body, inputs, outputs, nil)
+	body.SetHelp(fmt.Sprintln("This node filters ‘input’ values. All ‘input’ values are passed to ‘output’ EXCEPT ‘input’ values which are equal to ‘match’. Output the last known input string value if two inputs equal to each other. Out put nil if any of the inputs are nil."))
+	return &PreventEqualString{body, "defaultString"}, nil
+}
+
+func NewOnlyBetween(body *node.Spec) (node.Node, error) {
+	body = node.Defaults(body, onlyBetween, category)
+	inputs := node.BuildInputs(node.BuildInput(node.In, node.TypeFloat, nil, body.Inputs), node.BuildInput(node.InMin, node.TypeFloat, nil, body.Inputs), node.BuildInput(node.InMax, node.TypeFloat, nil, body.Inputs))
+	outputs := node.BuildOutputs(node.BuildOutput(node.Out, node.TypeFloat, nil, body.Outputs))
+	body = node.BuildNode(body, inputs, outputs, nil)
+	body.SetHelp(fmt.Sprintln("This node filters ‘input’ values. Only Numeric ‘input’ values between ‘min’ and ‘max’ are passed to ‘output’. Output the last known input float value if the input is not in range. Out put nil if any of the inputs are nil."))
+	return &OnlyBetween{body, 0}, nil
+}
+
+func NewOnlyGreater(body *node.Spec) (node.Node, error) {
+	body = node.Defaults(body, onlyGreater, category)
+	inputs := node.BuildInputs(node.BuildInput(node.In, node.TypeFloat, nil, body.Inputs), node.BuildInput(node.InMin, node.TypeFloat, nil, body.Inputs))
+	outputs := node.BuildOutputs(node.BuildOutput(node.Out, node.TypeFloat, nil, body.Outputs))
+	body = node.BuildNode(body, inputs, outputs, nil)
+	body.SetHelp(fmt.Sprintln("This node filters ‘input’ values. Only Numeric ‘input’ values greater than ‘OutMax’ are passed to ‘output’. Output the last known input float value if the input is less than 'OutMax'. Out put nil if any of the inputs are nil."))
+	return &OnlyGreater{body, 0}, nil
+}
+
+func NewOnlyLower(body *node.Spec) (node.Node, error) {
+	body = node.Defaults(body, onlyLower, category)
+	inputs := node.BuildInputs(node.BuildInput(node.In, node.TypeFloat, nil, body.Inputs), node.BuildInput(node.InMax, node.TypeFloat, nil, body.Inputs))
```

Comment:
correct input names to match rubix-wires documentation (ie. `threshold` not `in-max`). Leave output names as `node.Out`.